### PR TITLE
testing Coverall

### PR DESCRIPTION
**Pull request by aviehu**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -9,3 +9,9 @@ jobs:
       - run:
           name: eslint check
           command: yarn lint
+      - run:
```
Comment (roeeyud) [inline .circleci/config.yml]: no need to run tests twice...

File: package.json [hunk 2]
```diff
@@ -7,6 +7,8 @@
         "packages/*"
     ],
     "scripts": {
-        "lint": "eslint ./"
+        "lint": "eslint ./",
+        "test": "jest --coverage --coverageReporters=text-lcov | coveralls",
```
Comment (roeeyud) [inline package.json]: it looks like both scripts are doing coverage... why?

File: packages/server/package.json [hunk 3]
```diff
@@ -6,7 +6,9 @@
     "dependencies": {
         "babel-core": "^6.26.3",
         "babel-preset-env": "^1.7.0",
+        "babel-preset-es2015": "^6.24.1",
         "babel-register": "^6.26.0",
+        "coveralls": "^3.0.2",
```
Comment (roeeyud) [inline packages/server/package.json]: should be devDependency

File: README.md [hunk 4]
```diff
@@ -1,3 +1,5 @@
+<a href='https://coveralls.io/github/Home-Schooled/COQ?branch=coverall'><img src='https://coveralls.io/repos/github/Home-Schooled/COQ/badge.svg?branch=coverall' alt='Coverage Status' /></a>
```
Comment (roeeyud) [inline README.md]: Not sure I like this... 
it's cool and all but it doesn't match how I want branches to work.
You can add a ticket in trello to add this feature for master branch for later

Comment (aviehu) [reply README.md]: so for now just remove this and well come back to it?

**Pull request merged**